packages/core/src/components/popover/_popover.scss

```diff
@@ -120,6 +120,10 @@ $popover-width: $pt-grid-size * 35 !default;
   .pt-transition-container {
     @include popover-inline-position();
   }
+
+  .pt-overlay-inline {
```

Comment:
can you add a comment why this is necessary? so we don't come wondering later. might require re-deriving why we had to add this.